pkg/karmadactl/get_flags.go

```diff
@@ -292,55 +275,6 @@ func (f *ConfigFlags) ToRESTMapper() (meta.RESTMapper, error) {
 	return expander, nil
 }
 
-// AddFlags binds client configuration flags to a given flagset
-func (f *ConfigFlags) AddFlags(flags *pflag.FlagSet) {
-	if f.KubeConfig != nil {
-		flags.StringVar(f.KubeConfig, "kubeconfig", *f.KubeConfig, "Path to the kubeconfig file to use for CLI requests.")
-	}
-	if f.CacheDir != nil {
-		flags.StringVar(f.CacheDir, flagCacheDir, *f.CacheDir, "Default cache directory")
-	}
-
-	// add config options
-	if f.CertFile != nil {
-		flags.StringVar(f.CertFile, flagCertFile, *f.CertFile, "Path to a client certificate file for TLS")
-	}
-	if f.KeyFile != nil {
-		flags.StringVar(f.KeyFile, flagKeyFile, *f.KeyFile, "Path to a client key file for TLS")
-	}
-	if f.BearerToken != nil {
-		flags.StringVar(f.BearerToken, flagBearerToken, *f.BearerToken, "Bearer token for authentication to the API server")
-	}
-	if f.ClusterName != nil {
-		flags.StringVar(f.ClusterName, flagClusterName, *f.ClusterName, "The name of the kubeconfig cluster to use")
-	}
-	if f.AuthInfoName != nil {
-		flags.StringVar(f.AuthInfoName, flagAuthInfoName, *f.AuthInfoName, "The name of the kubeconfig user to use")
-	}
-	if f.Namespace != nil {
-		flags.StringVarP(f.Namespace, flagNamespace, "n", *f.Namespace, "If present, the namespace scope for this CLI request")
-	}
-	if f.Context != nil {
-		flags.StringVar(f.Context, flagContext, *f.Context, "The name of the kubeconfig context to use")
-	}
-
-	if f.APIServer != nil {
-		flags.StringVarP(f.APIServer, flagAPIServer, "s", *f.APIServer, "The address and port of the Kubernetes API server")
-	}
-	if f.TLSServerName != nil {
-		flags.StringVar(f.TLSServerName, flagTLSServerName, *f.TLSServerName, "Server name to use for server certificate validation. If it is not provided, the hostname used to contact the server is used")
```

Comment:
The description of these flags seem useful for us to understand them, what do you think? @carlory